test/runtests.jl

```diff
@@ -237,51 +221,52 @@ include("geojson_samples.jl")
             @test GI.trait(f1) === GI.FeatureTrait()
             @test GI.geomtrait(geom) === GI.MultiPolygonTrait()
             properties = GeoJSON.properties(f1)
-            @test properties isa JSON3.Object
-            @test properties["addr2"] === "Rowland Heights"
-            @test !GI.isclosed(GeoJSON.read(T.bbox))
-            @test GI.isclosed(GeoJSON.read(T.bermuda_triangle))
+            @test properties isa Dict
+            @test properties[:addr2] === "Rowland Heights"
+            @test !GI.isclosed(GeoJSON.read(Samples.bbox))
+            @test GI.isclosed(GeoJSON.read(Samples.bermuda_triangle))
         end
     end
 
     @testset "Tables with missings" begin
-        t = GeoJSON.read(T.tablenull)
+        t = GeoJSON.read(Samples.tablenull)
         @test t[1] isa GeoJSON.Feature
         @test occursin("(:geometry, :a, :b)", sprint(show, MIME"text/plain"(), t[1]))
         @test ismissing(t[1].geometry)
         GeoJSON.geometry(t[1])
         @test t.geometry isa Vector{Union{T,Missing}} where {T<:GeoJSON.Point}
         @test ismissing(t.geometry[3])
-        @test t.a isa Vector{Union{Float64,Missing}}
+        @test t.a isa Vector{Union{Int64,Missing}}
         @test isequal(t.a, [1, missing, 3])
         @test t.b isa Vector{Missing}
         @test Tables.columntable(t) isa NamedTuple
 
-        t = GeoJSON.read(T.table_not_present)
+        t = GeoJSON.read(Samples.table_not_present)
         @test occursin("(:geometry, :a, :b, :c)", sprint(show, MIME"text/plain"(), t[1]))
-        @test propertynames(t) == [:geometry, :a, :b, :c, :d]
+        @test propertynames(t) == [:a, :b, :c, :geometry, :d]
         @test propertynames(t[1]) == (:geometry, :a, :b, :c)
         @test propertynames(t[2]) == (:geometry, :a, :b, :d)
         # "c" and "d" are only present in the properties of a single row
         @test all(t.c .=== ["only-here", missing, missing])
         @test all(t.d .=== [missing, "appears-later", missing])
         @testset "With NamedTuple feature" begin
```

Comment:
Its not a NamedTuple now this is a bit confusing.

Does `NamedTuple` as properties not work now?

Reply:
Ok, now switched to NamedTuple for the fun of it. Can always revert.